frontend/vue/components/UtilityPanel/Scratchpad.vue

```diff
@@ -29,11 +29,11 @@
       <div v-if="isApiTokenNeeded" class="scratchpad__api-token-info">
         <WarningIcon />
         <div>
-          This code is executed on real hardware using an IBM Quantum provider. Setup your API-token in
+          This code looks like it connects to IBM Quantum. Add your API-token to
           <BasicLink url="/account/admin">
             your account
           </BasicLink>
-          to execute this code cell.
+          to run this cell.
```

Comment:
i would just run this by design (@JRussellHuffman , @gracelindsell) and confirm they are OK with the updated text.

Reply:
Sounds good, thanks. That message is just for the scratchpad (which only checks for `IBMQ.`, the message for the inline cells is a bit different:

> This code connects to, or relies on results from IBM Quantum. Add your API-token to your account to run this cell.

Reply:
What is the updated text you are considering? Is it the text in the screenshots above, or @frankharkins's quoted text directly above, or both?

Reply:
@JRussellHuffman 
For scratchpad:
> This code looks like it connects to IBM Quantum. Add your API-token to your account to run this cell.

For inline cells:
> This code connects to, or relies on results from IBM Quantum. Add your API-token to your account to run this cell.

The screenshot only shows the text for the inline cells.

Reply:
@JRussellHuffman @gracelindsell you good with the above text updates.

Reply:
yep. I like it.